app/views/subjects/show.html.haml

```diff
@@ -1,7 +1,9 @@
-%h1= @subject.name
-%h3
+%h1= "Subject: " + @subject.name
```

Comment:
For convention, ruby uses the string interpolation format instead of concatenating strings like other languages
```suggestion
%h1= "Subject: #{@subject.name}"
```